<?xml version="1.0"?>
<launch>

<arg name="Mavros" default="0"/>
<arg name="DJI" default="1"/>

    <!-- <include file="$(find dji_sdk)/launch/sdk.launch"/> -->

    <node name="wpts_node" pkg="swarmsim_example" type="swarmsim_example" output="screen">
        <param name="droneType" value="$(arg Mavros)"/>
    </node>
    
    <!--include file="$(find goal_publisher)/launch/goal_pub.launch"/-->


    <!-- <arg name="nDrones" default="1" />
    <arg name="predefined" default="false" />
    <arg name="yamlFileName" default="goals.yaml"/>
    <arg name="traj_dir" default="$(find swarmsim_example)/launch/traj_data/"/>

    <node name="swarmsim_example" pkg="swarmsim_example" type="wpts_node" output="screen">
        <param name="trajDir" value="$(arg traj_dir)"/>
        <param name="nDrones" value="$(arg nDrones)"/>
        <param name="predefined" value="$(arg predefined)"/>
        <param name="yamlFileName" value="$(arg yamlFileName)"/>
    </node> -->


    <!-- <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find swarmsim_example)/launch/custom_rosconsole.conf"/> -->

</launch>
